crates/ui/src/components/button/icon_button.rs

```diff
@@ -151,7 +151,7 @@ impl RenderOnce for IconButton {
     fn render(self, window: &mut Window, cx: &mut App) -> impl IntoElement {
         let is_disabled = self.base.disabled;
         let is_selected = self.base.selected;
-        let selected_style = self.base.selected_style;
+        let selected_style = self.base.selected_style.clone();
```

Comment:
Is this clone still necessary, or was it necessary earlier and no longer needs to be after other changes? 😄

Reply:
I had to make the selected style type not `Clone`, so still necessary :smile: